pytket/phir/sharding/shard.py

```diff
@@ -57,3 +58,8 @@ def pretty_print(self) -> str:
         content = output.getvalue()
         output.close()
         return content
+
+
+Cost: TypeAlias = float
+Layer: TypeAlias = list[Shard]
+Ordering: TypeAlias = list[int]
```

Comment:
Unsure if this file is the best place to place these alias definitions. I chose this as both `phirgen` and `place_and_route` already imported it.

Happy to take suggestions. One option is to create a new module with all type declarations.